### PR TITLE
Update azure_rm_route related document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_route related document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_route/azure_rm_routetable/azure_rm_routetable_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_routetable.py [hunk 1]
```diff
@@ -73,13 +72,17 @@
 '''
 RETURN = '''
 changed:
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_routetable.py:74]: While we're updating this documentation, I would remove the entire entry for `changed` from the `RETURN` section - `changed` is not returned by the module, but by Ansible core, so it doesn't really belong in the module documentation.

File: lib/ansible/modules/cloud/azure/azure_rm_routetable.py [hunk 2]
```diff
@@ -73,13 +72,17 @@
...
-    description: Whether the resource is changed.
+    description:
+        - Whether the resource is changed.
     returned: always
     type: bool
+    sample: true
 id:
-    description: resource id.
+    description:
+        - Resource id.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_routetable.py]:
```suggestion
        - Resource ID.
```

File: lib/ansible/modules/cloud/azure/azure_rm_routetable_facts.py [hunk 3]
```diff
@@ -61,37 +61,49 @@
 '''
 RETURN = '''
 id:
-    description: Resource id.
+    description:
+        - Resource id.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_routetable_facts.py]:
```suggestion
        - Resource ID.
```

**Review by acozine — COMMENTED** [review]
Thanks @Fred-sun. Overall looks good. Thanks for adding the sample values to the returns - that's very helpful.

As usual, I found a few small suggestions.

File: lib/ansible/modules/cloud/azure/azure_rm_routetable.py (hunk 1 above)
Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_routetable.py:74]: Thank you for your review. I will pay attention to this problem in future maintenance. Thanks a lot!

**Comment by Fred-sun** [comment]
ready_for_review

**Pull request merged**